src/main.rs

```diff
@@ -117,10 +39,8 @@ pub fn main() {
             eprintln!("Error during startup: {}", e);
             error!("Error during startup: {}", e);
             match e.raw_os_error() {
-                Some(13) => {
-                    startup_helpers::report_permission_error();
-                },
-                _ => eprintln!("Something else!")
+                Some(13) => report_permission_error(),
+                _        => eprintln!("Something else!")
```

Comment:
I think we should just remove this and let it be {}. Printing "something else" is confusing without context.

Reply:
It seems like you've done this, so I am resolving this conversation.